handbook/operations.md

```diff
@@ -0,0 +1,63 @@
+# Operations
+
+## People Ops
+
+### Hiring a new team member
+
+1. Reach out to Mike and Zach and give them an opportunity to meet the candidate.
+
+2. Research compensation using [Pave](https://www.pave.com), making sure to adjust for geographical factors. *If you're unsure of your findings, ask for help from Mike.*
+
+3. Share the compensation information you've gathered with Eric. Eric will confirm all this information with Mike and document this decision in the [compensation decisions document](https://docs.google.com/document/d/1NQ-IjcOTbyFluCWqsFLMfP4SvnopoXDcX0civ-STS5c/edit) for future reference.
+
+4. After compensation has been determined, you need to create two documents for the candidate, an "exit scenarios" spreadsheet and an "informal offer email" template. If you need these templates, reach out to Eric on slack, and he will provide you links that automatically create copies Change the name of the documents accordingly (e.g., "[candidate's name]'s copy of exit scenarios,") and link to the exit scenarios spreadsheet from the offer email.
```

Comment:
```suggestion
4. After compensation has been determined, create two documents for the candidate:

- "Exit scenarios" spreadsheet
- "Informal offer email" template. 

If you need these templates, reach out to Eric on Slack, and he will provide you links that automatically create copies. 

Change the name of the documents accordingly (e.g., "[candidate's name]'s copy of exit scenarios,") and link to the exit scenarios spreadsheet from the offer email.
```

Reply:
Point 4 implies that you will need templates for "Exit scenarios" and "Informal offer email", but the following text suggests "IF you need these templates, reach out to Eric for a link". Can we remove any confusion/extra admin for all involved and just link to the templates from here?